src/services/getTrackData.js

```diff
@@ -0,0 +1,3 @@
+export default function getTrackData() {
+  return fetch('http://localhost:4000/tracks').then((res) => res.json())
```

Comment:
I would recommend some error catching if server is not available.

Reply:
> Looks great! You probably have a separate project for the server you wrote, right? Does that also need to be reviewed?

I have a json server installed in my react app. You can find the data in "src / data / db.json"